packages/frontend/core/src/components/blocksuite/block-suite-header/menu/index.tsx

```diff
@@ -83,6 +87,14 @@ export const PageHeaderMenuButton = ({
     return setOpenHistoryTipsModal(true);
   }, [setOpenHistoryTipsModal, workspace.flavour]);
 
+  const setOpenInfoModal = useSetAtom(openInfoModalAtom);
+  const openInfoModal = () => {
+    mixpanel.track('Button', {
+      resolve: 'OpenDocInfoModal',
+    });
```

Comment:
please confirm the payload with the product team

Reply:
I will remove them first, and wait for the discussion to create another PR about buried points.

Reply:
"buried points"